metadata/namespaces.go

```diff
@@ -27,6 +27,8 @@ import (
 	bolt "go.etcd.io/bbolt"
 )
 
+const sharedNamespaceLabel = "containerd.io/namespace/sharable"
```

Comment:
If this label is to be a public interface, it should be exported somewhere

not sure what the best location would be for it. I see we have one defined in `labels/labels.go`, but perhaps there's other / better locations https://github.com/containerd/containerd/blob/f0a32c66dad1e9de716c9960af806105d691cd78/labels/labels.go#L21

Reply:
I'm going to go ahead and move it here, and slightly change the name to be consistent with this format.